test/ui-test/src/screens/StatusMainScreen.py

```diff
@@ -38,7 +38,8 @@ def joinChatRoom(self, room: str):
         type(ChatNamePopUp.INPUT_ROOM_TOPIC_TEXT.value, room)
         click_obj_by_name(ChatNamePopUp.START_CHAT.value)
         
-    
+    def goToCommunityPortal(self):
```

Comment:
nit, keep same naming/convention: `open_community_portal`

Reply:
> nit, keep same naming/convention: `open_community_portal`

Yes, please, update the method name ;)

Reply:
> nit, keep same naming/convention: `open_community_portal`

Yes, please, could you update the method name?! ;)

Reply:
updated